.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: lint
```

Comment:
Out of curiosity, can you try also renaming this file to `lint.yml` to see if it'll trigger a run?

Reply:
A run was triggered with the file being named lint.yml. (called lint / lint)

Reply:
sweet, thanks for testing that out!